lib/diagnostic.rb

```diff
@@ -105,15 +123,25 @@ def eat_meat(food)
 end
 
 ##
-# your response here
+class Lion < Cat
+include Carnivorous
+def initialize (roar)
```

Comment:
This is a constructor.  We want an instance method named `roar` that prints `ROAR!` to the console.

```ruby
class Lion < Cat
  include Carnivorous

  def roar
    puts 'ROAR!'
  end
end
```